tests/integration/components/frost-checkbox-test.js

```diff
@@ -248,7 +245,7 @@ describeComponent(
     })
 
     // https://github.com/juwara0/ember-frost-core/issues/1
-    it.skip('sets focus on render when autofocus is true', function () {
+    it('sets focus on render when autofocus is true', function () {
```

Comment:
The issue that is causing this test to fail is not fixed yet: https://github.com/ciena-frost/ember-frost-core/issues/257 

This test was skipped until the issue was fixed.